TRIAGING.md

```diff
@@ -0,0 +1,118 @@
+
+The GitHub Actions in the `conda/infra` repository are viewed as canonical; the [Sync workflow](https://github.com/conda/infra/blob/main/.github/workflows/sync.yml) sends out any modifications to [other `conda` repositories](https://github.com/conda/infra/blob/main/.github/sync.yml#L3-L17) from there.
+
+
+### What is done about the issues in "triaging" mode?
+
+Issues in the "Triaging" column of the board have been reviewed by a triaging engineer and are considered ready for the following procedures:
+
+- Mitigation via short-term workarounds and fixes
+- Redirection to the correct project
+- Determining if support can be provided for errors and questions
+- Closing out of any duplicate/off-topic issues
+
+The triaging engineer on rotation is not seeking to _resolve_ issues that arise. Instead, the goal is to understand the issue and to understand whether it is an issue in the first place and to then collect as much relevant information as possible so that the maintainers of `conda` can make an informed decision on the appropriate resolution schedule necessary.
+
+Issues will remain in the "Triaging" column as long as the issue is in an investigatory phase (_e.g._, querying the user for more details, asking the user to attempt other workarounds, other debugging efforts, etc.) and are likely to remain in this state the longest but should still be progressing over the course of 1-2 weeks.
+
+
+### When do items move out of "Triaging"?
+
+The additional columns in the triaging board that the issues can be moved to include the following:
+
+- **"Support"** - Any issue moved into this column is a request for support and is not a feature request or a bug report.
+- **"Ready"** - The issue has revealed a bug or feature request. We have collected enough details to understand the problem/request and to reproduce it on our own. These issues are ready to be moved into the Backlog at the end of the triage rotation during Refinement.
+- **"Closed"** - The issue was closed due to being a duplicate, being redirected to a different project, was a user error, a question that has been resolved, etc.
+
+
+### Where do items go after being triaged?
+
+All triaged issues will be reviewed by triaging engineers during a weekly Refinement meeting in order to understand how the triaged issues fit into the short- and long-term roadmap of `conda`. These meetings enable the triaging engineers to get together to decide the prioritization of issues; the outcome of these discussions are made publicly available via [GitHub Milestones](https://github.com/conda/conda/milestones). It will also be during Refinement where decisions are made to earmark feature requests for specific future releases (versus a more open-ended backlog), tag issues as ideal for first-time contributors, as well as whether or not to close/reject specific feature requests.
```

Comment:
We haven't made any policy decisions ATM for when and where we use milestones so best to remove that.

```suggestion
All triaged issues will be reviewed by triaging engineers during a weekly Refinement meeting in order to understand how the triaged issues fit into the short- and long-term roadmap of `conda`. These meetings enable the triaging engineers to get together to collectively prioritize issues, earmark feature requests for specific future releases (versus a more open-ended backlog), tag issues as ideal for first-time contributors, as well as whether or not to close/reject specific feature requests.
```